app/main.py

```diff
@@ -1,8 +1,50 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self, distance_from_city_center: int,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: float) -> None:
+
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+        if distance_from_city_center < 1.0:
+            raise ValueError("Distance from city center must be above 1.0")
```

Comment:
The condition `distance_from_city_center < 1.0` should be adjusted to `distance_from_city_center <= 0` to allow a minimum distance of 1.0, as the error message suggests that 1.0 is acceptable.